ojd_daps_skills/analysis/OJO/streamlit_viz/README.md

```diff
@@ -0,0 +1,20 @@
+# Streamlit app
+
+We created a visualistion in Streamlit in order for others to explore the dataset.
+
+The first step in creating this visualisation is preparing the data for it. This is done by running:
+
+```
+python ojd_daps_skills/analysis/OJO/process_viz_data.py
+```
+
+[This requires the data saved by running `python ojd_daps_skills/analysis/ojo/get_skill_occurrences_matrix.py`]
+
+## In progress
+
+A preliminary analysis app can be seen by running:
+
+```
+pip install -r analysis_app_requirements.txt
```

Comment:
I think this needs adding to the PR

Reply:
I also think the standard is `requirements_analysis_app.txt` rather than using it as a suffix. If its easy to change I think thatd look nicer - but if its a pain dont worry